### ISSUE TITLE
Parameter Identification: erroneous behavior of "Transfer results to Simulation" in combination with "Use as factor"

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.Core#2754]
PK-Sim v7.1.0 Build 50.
When the option "Use as factor" is selected for an identification parameter and identified results are transfered to simulation, not the values are transfered but the actual value in the simulation is multiplied by the identified factor. The behavior has major drawbacks:
1) Multiple selecting the "Transfer to Simulation"-button results in sequential multiplication of the parameter value
2) If the value of the parameter was changed in the simulation while PI was running, "Transfer to simulation" produces values different from identified ones.

Examlpe:
1) Open the "Theophylline" project.
2) Select simulation "Kaumeier IV 208 mg fit" for parameter identification.
3) Select "Lipophilicity" as identification parameter and check "Use as factor".
4) Start value 0.5, min value 0.49, max value 0.51.
5) Run the PI.
6) Identified value (as factor) is 0.51. Transfer it to simulation. The value of lipophilicity changes to 0.45 Log Units.
7) Transfer PI results once more. The value of lipophilicity is 0.23.

**Comment by rwmcintosh** [comment]
I'm not sure if this transfer is common to MoBi but both apps need to take this into account if it's not implemented in core.

**Comment by PavelBal** [comment]
> I'm not sure if this transfer is common to MoBi

It is.

**Comment by rwmcintosh** [comment]
In either case (factor or not factor) can we make this button a one-shot that is disabled after it's used once then re-enabled when a PI runs again?

**Comment by rwmcintosh** [comment]
Or we capture the value of the linked parameter when the PI starts and we can only apply the factor if the current parameter value is the original value.

**Comment by PavelBal** [comment]
> In either case (factor or not factor) can we make this button a one-shot that is disabled after it's used once then re-enabled when a PI runs again?

No, not a good solution. I might change the simulation manually, and would like to apply estimated parameter values again.

**Comment by rwmcintosh** [comment]
Ok, for that to work we need to have the initial value of the parameter stored in the PI and the factor can only be applied to the initial value and not to the current value.
